### ISSUE TITLE
Ip Address validation is giving wrong result even if ip address is in range of given network

### ISSUE THREAD

**Issue by ReetaWani** [seancfoley/IPAddress#57]
Hi @seancfoley , I have used this library, to validate the CIDR range in my spring boot application. I Observed it validates the ip address correctly if I provide 10.0.0.0/8 as network and 10.168.89.136 address.

`contains("10.0.0.0/8", "10.168.89.136"); -- True`
`contains("10.10.10.0/8", "10.168.89.136"); -- False`

Can you tell me whether it is intended behavior or i'm missing something

**Comment by seancfoley** [comment]
This a duplicate of issues #26, issue #30,  issue #45, issue #51, issue #54 and also similar to issue #39 and issue #53.

This is explained in [this wiki entry](https://github.com/seancfoley/IPAddress/wiki/Code-Examples-1:-Creating,-Parsing,-Checking-Host-Address-Subnet,-Extracting-Address-Subnet-Info#parse-prefixed-ip-address-string-as-subnet-or-ip-address) and this [entry](https://github.com/seancfoley/IPAddress/wiki/Code-Examples-2:-Subnet-Containment-and-Matching-Checks#check-if-subnet-contains-address-or-subnet) - see enclosingBlockContains() in particular.

In summary, 
10.10.10.0/8 is the same as 10.10.10.0, just a single address, because the host is not zero.  10.0.0.0/8 is a subnet of many addresses because the host is zero.  If you want to convert to the subnet, then use toPrefixBlock.

    new IPAddressString("10.10.10.0/8").getAddress().toPrefixBlock().
        contains(new IPAddressString("10.10.10.0").getAddress()); //true
